web/client/codechecker_client/cmd/store.py

```diff
@@ -467,6 +467,11 @@ def assemble_zip(inputs, zip_file, client):
         if file_hashes else []
     LOG.info("Get missing file content hashes done.")
 
+    LOG.info("Get missing blame file content hashes from the server...")
```

Comment:
```suggestion
    LOG.info("Get file content hashes which do not have blame information from the server...")
```
or something similar.